src/auth.ts

```diff
@@ -1,8 +1,48 @@
 import AinftBase from './ainftBase';
+import { INITIALIZE_STAKE_AMOUNT } from './constants';
 import { HttpMethod, User } from './types';
-
 export default class Auth extends AinftBase {
 
+  async initializeApp(appId: string, userId: string): Promise<void> {
+    const accessAccount = this.ain.wallet.defaultAccount!;
+    const createAppRes = await this.createApp(appId, userId, accessAccount.address);
+    console.log(`create app tx hash - ${createAppRes.txHash}`);
+    const stakeRes = await this.stake(appId, userId, INITIALIZE_STAKE_AMOUNT);
+    console.log(`stake tx hash - ${stakeRes.txHash}`);
+    const setRuleRes = await this.setBlockchainActivityRule(appId);
+    console.log(`set rule tx hash - ${setRuleRes.txHash}`);
+  }
+
+  createApp(appId: string, userId: string, accessKey: string) {
+    const body = { appId, userId, accessKey };
+    const trailingUrl = 'create_app';
+    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+  }
+
+  stake(appId: string, userId: string, amount: number) {
+    const body = { appId, userId, amount };
+    const trailingUrl = 'stake';
+    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+  }
+
+  createUser(appId: string, userId: string): Promise<User> {
+    const body = { appId, userId };
+    const trailingUrl = 'create_user_account';
+    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+  }
+
+  createAdmin(appId: string, userId: string): Promise<User> {
```

Comment:
question) what happens when the user is already created but not admin yet?

Reply:
This method creates a new user and registers it as an admin.  
A new method is required to register an existing user as an admin.

Reply:
Do we have any use cases already?
If not, can we add addAdminUser() and removeAdminUser() instead?

Reply:
I'll work on the next PR!

Reply:
👍  could you add a todo?

Reply:
done!